relay/kusama/src/lib.rs

```diff
@@ -920,7 +923,8 @@ impl claims::Config for Runtime {
 parameter_types! {
 	// Minimum 100 bytes/KSM deposited (1 CENT/byte)
 	pub const BasicDeposit: Balance = 1000 * CENTS;       // 258 bytes on-chain
-	pub const FieldDeposit: Balance = 250 * CENTS;        // 66 bytes on-chain
+	// TODO:(PR#137) - check ByteDeposit?
+	pub const ByteDeposit: Balance = deposit(0, 1);
```

Comment:
Correct by definition, but you can add a sanity check test if you want.

Reply:
@georgepisaltu can verify

Reply:
It is the cost per byte stored, as can be seen [here](https://github.com/paritytech/polkadot-sdk/blob/13f2342edd70d83377d620385610bc8dc859b64b/substrate/frame/identity/src/lib.rs#L1240) and [here](https://github.com/paritytech/polkadot-sdk/blob/13f2342edd70d83377d620385610bc8dc859b64b/substrate/frame/identity/src/lib.rs#L1386).